### ISSUE TITLE
The papar doesn't use the sigmoid function.

### ISSUE THREAD

**Issue by dongdong092** [godisboy/SN-GAN#7]
While in your code,i see the sigmoid function in the last layer of the discrimator. I've noticed that you may have try the model with the linear layer instead of the sigmoid function.Does it work well?

**Comment by godisboy** [comment]
Hi @dongdong092 
In the original paper, the author had claimed that they used the standard objective function for GAN.
![image](https://user-images.githubusercontent.com/14243507/40285761-d440e50c-5cda-11e8-88b3-72353a4df9b6.png)

**Comment by dongdong092** [comment]
But with Cross entropy  it will become JS divergence again, why not use WGAN's method to use D(X) as the loss function directly?

**Comment by zhangqianhui** [comment]
@dongdong092  It use the original loss: -log d(sigmoid(x)) = softplus(-x). You can use wgan loss. But it may not get better results.
